tests/helper_functions.py

```diff
@@ -221,20 +221,18 @@ def __init__(self, proc, port, activity_polling_timeout_seconds=250):
         self._activity_polling_timeout_seconds = activity_polling_timeout_seconds
 
     def __enter__(self):
-        for i in range(0, int(self._activity_polling_timeout_seconds / 5)):
```

Comment:
Can I push a commit to this file?

Reply:
Yes, or if you what to put the change here i will get it!

Reply:
Can we replace print functions with logging methods as you suggested?

Reply:
Done, I put it with INFO category, is it ok like this or should I put another level?

Reply:
`INFO` sounds good to me.